tests/lib/rules/terminating-properties.js

```diff
@@ -50,5 +50,18 @@ ruleTester.run('terminating-properties', rule, {
     errors: [{
       message: '"to.exist" used as function'
     }]
+  },  {
+    options: [{terminators:['something', 'somethingElse']}],
```

Comment:
could you change the name of that option to `properties` instead? that would match what I proposed in #20

Reply:
I have done that, what do you think?

Reply:
sorry, I missed that